src/crypto/chacha20.cpp

```diff
@@ -146,6 +152,25 @@ void ChaCha20::Output(unsigned char* c, size_t bytes)
         x14 += j14;
         x15 += j15;
 
+        if (m != nullptr) {
+            x0 = XOR(x0, ReadLE32(m + 0));
```

Comment:
Not sure what the benefit of defining XOR macro is. Isn't `x0 ^= ReadLE32(m + 0)` more readable and less error-prone than `x0 = XOR(x0, ReadLE32(m + 0))`?

Reply:
Thanks for the review! Your right. Changed.